components/DismissibleMessage.js

```diff
@@ -30,9 +30,11 @@ const editAccountSettingsMutation = gqlV2`
  *
  * Messages will never be displayed if user is not logged in.
  */
-const DismissibleMessage = ({ LoggedInUser, messageId, children }) => {
+const DismissibleMessage = ({ LoggedInUser, messageId, displayForLoggedOutUser, children }) => {
   const settingsKey = `${DISMISSABLE_HELP_MESSAGE_KEY}.${messageId}`;
-  const [isDismissedLocally, setDismissedLocally] = React.useState(false);
+  const [isDismissedLocally, setDismissedLocally] = React.useState(
+    typeof window !== 'undefined' && window.localStorage.getItem(settingsKey) === 'true',
```

Comment:
Check `lib/local-storage.js` it has helpers to do that safely & cross-browser

```suggestion
      getFromLocalStorage(settingsKey) === 'true';
```

Reply:
Nice! Thanks!